share/completions/reg.fish

```diff
@@ -1,4 +1,24 @@
+function reg # placeholder for testing with wine, will be removed at the end of PR
+    wine reg $argv 2> /dev/null
+end
+
+function __reg_run_reg_safely
+    #if which reg > /dev/null # these lines will be uncommented at the end of PR
+        set -l output (reg $argv | tr -d '\r' | tail --lines +2 | string collect)
+        if not string match -q -r "reg: Invalid syntax*" -- $output
+            set output (echo $output | string replace " " "\n" | string collect)
```

Comment:
This is just `set output (string split \n -- $output)`.

Or, rather... It's just *not* doing the `string collect` above to begin with.

Just keep the output split per line, and check the first line with that `string match` (`$output[1]`).